README.md

```diff
@@ -1,2 +1,64 @@
-# tw-academy-s9
-Homework repository
+# Sample Style Guide
+Technical Writing Academy - May 2021
+
+## Bold
+
+Use bold to highlight button names, field names, menus, radio buttons, or checkboxes. For example:
+* Click the **create** button to start the process.
+* Enter the IP adress in the **IP adress** field.
+
+## Italics
+
+Use italics to emphasize a word or a group of words. For example:
+* *At least two* reviews are requiered before you check in your code.
+* You must *not* enter without your ID card.
+
+## Fixed Font
+Use a fixed font to highlight:
+* **Commands or pieces of code.** For example: Run the `npm install text-cli` command to install the utiliy.
+* **File or folder names.** For example: Read the `README.md` file before you proceed.
+
+## Abbreviations
+Never use the following abbrevations in a formal document:
+* e.g. - instead use "for example."
+* i.e. - instead use "in other words."
+* etc. - instead give the full list of options
+* fig. - instead spell it out as "figure."
+* pg. - instead spell it out as "page."
+
+## Capitalization
+Use the standard capitalization rules. Also, keep in mind the following:
+* The first word after a colon (:) is always capitalized.
+* Every heading, table caption, figure caprion, columns name, or callout in a figure or table must be in title case. 
+
+## Acronyms
+Always define an acronym when you are using it for the first time. For example: This section specifies the terms used in a Product Requirement Document (PRD). The most often used terms in a PRD include...
+
+## Commas in Lists
+The ruler regarding commas for in-sentence lists are as follows:
+* Do not use a comma before "and" and "or" if your list contains only two items. For example:
+Any new organization needs to invest in good infrastructure and hiring great talent.
+* Use a comma before "and" and "or" when your list comprises three or more items. For example:
+A "Pod" comprises a Project Manager, a Technical Writer, Engineers, and UX designers.
+
+## Numbers 
+Remember these rules about numbers:
+* Spell out numbers that are less than 10 (0 - 9). <br>
```

Comment:
The br tags in this section are not causing any problems in the rendered markdown. However, to have a cleaner file and prevent future issues it might be a good idea to remove them.

Reply:
Ok, I included them to have the same format as the guide (we sometimes do that at Dow Jones 🤔). I am erasing them from this doc.

Reply:
I see. They shouldn't be necessary for regular markdown like this, but I agree that with the Dow Jones pages sometimes it is necessary to achieve the correct output.